src/components/Header.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { ReactComponent as Logo } from './logo.svg';
+import Nav from './Nav';
+
+function Header() {
+  return (
+    <header>
+      <Link className="logo" to="/"><Logo /></Link>
+      <Nav />
```

Comment:
I wouldn't separate the Header and Nav components. It feels like unnecessary abstraction since they both are very simple.

If you decide to leave them separate I'd suggest to create a `src/components/Header` directory and move the Header and Nav components there. The Nav component will only be used inside the Header component and thus it makes sense to keep them together